packages/agent/package.json

```diff
@@ -4,7 +4,15 @@
   "author": "DFINITY Stiftung <sdk@dfinity.org>",
   "license": "Apache-2.0",
   "description": "JavaScript and TypeScript library to interact with the Internet Computer",
-  "homepage": "https://sdk.dfinity.org/docs/index.html",
+  "homepage": "https://smartcontracts.org",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dfinity/agent-js.git",
+    "directory": "packages/agent"
```

Comment:
I don't think the directory config is correct - this package.json already lives inside packages/agent

Reply:
As far as I understand, the `directory` refers just to the repo given by the `url` above, i.e. the base repo, and has nothing to do with the `package.json`.

Reply:
@krpeacock